model/atmosphere/dycore/tests/dycore_tests/test_solve_nonhydro.py

```diff
@@ -850,10 +815,10 @@ def test_run_solve_nonhydro_single_step(
 @pytest.mark.datatest
 @pytest.mark.parametrize("experiment", [dt_utils.REGIONAL_EXPERIMENT])
 @pytest.mark.parametrize(
-    "istep_init, jstep_init, step_date_init, istep_exit, jstep_exit, step_date_exit, vn_only",
+    "istep_init, jstep_init, step_date_init, istep_exit, jstep_exit, step_date_exit",
     [
-        (1, 0, "2021-06-20T12:00:10.000", 2, 1, "2021-06-20T12:00:10.000", False),
-        (1, 0, "2021-06-20T12:00:20.000", 2, 1, "2021-06-20T12:00:20.000", True),
```

Comment:
I must have removed it by mistake, I think it is used by 

```python

@pytest.fixture
def savepoint_velocity_exit(data_provider, step_date_exit, istep_exit, vn_only, jstep_exit):  # F811
    """
    Load data from ICON savepoint at exist of solve_nonhydro module.

    date of the timestamp to be selected can be set seperately by overriding the 'step_data'
    fixture, passing 'step_data=<iso_string>'
    """
    return data_provider.from_savepoint_velocity_exit(
        istep=istep_exit, vn_only=vn_only, date=step_date_exit, jstep=jstep_exit
    )
```

for example. I've added it back in.

Reply:
Thank you. It might still be that the `istep`, `jstep`and `step_date_exit` determine the savepoint uniquely and `vn_only` is redundant, but I don't remember.